interfaces/Portalicious/src/app/pipes/translatable-string.pipe.ts

```diff
@@ -0,0 +1,36 @@
+import { LOCALE_ID, Pipe, PipeTransform, inject } from '@angular/core';
+import { Locale } from '~/utils/locale';
+
+@Pipe({
+  name: 'translatableString',
+  standalone: true,
+})
+export class TranslatableStringPipe implements PipeTransform {
```

Comment:
The `TranslatableStringPipe` should work the same as the `TranslatableStringService`.

Only 1 additional feature: If there is not `en`-label available, it will take any other language at least... 
(See the issue with some French-only programs on the Training/Demo-environment)